.github/workflows/e2e-functional-template.yml

```diff
@@ -149,6 +149,9 @@ jobs:
           wget -qO - https://download.opensuse.org/repositories/Emulators:/Wine:/Debian/xUbuntu_22.04/Release.key | sudo apt-key add -
           wget -qO /usr/local/bin/yq https://github.com/mikefarah/yq/releases/download/v4.20.1/yq_linux_amd64 && chmod a+x /usr/local/bin/yq
           sudo apt-get update || true && sudo apt-get install -y ca-certificates libxtst-dev libpng++-dev gcc-aarch64-linux-gnu g++-aarch64-linux-gnu jq icnsutils graphicsmagick tzdata xsel
+          sudo apt-get install -y xvfb
```

Comment:
adding this to prevent electron crashing when test runs

Reply:
What does this do?

Reply:
Hi @devinbinnie 
this is explicitly installed, so that in headless mode the e2e tests have a display and run graphical operations in memory.